scripts/configure_dev.sh

```diff
@@ -43,38 +49,40 @@ function install_or_upgrade {
 function install_windows_shellcheck() {
     version="v0.7.1"
     if ! wget https://github.com/koalaman/shellcheck/releases/download/$version/shellcheck-$version.zip -O /tmp/shellcheck-$version.zip; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Error downloading shellcheck $version"
         return 1
     fi
 
     if ! unzip -o /tmp/shellcheck-$version.zip shellcheck-$version.exe -d /tmp; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Unable to decompress shellcheck $version"
         return 1
     fi
 
     if ! mv -f /tmp/shellcheck-$version.exe /usr/bin/shellcheck.exe; then
-        rm /tmp/shellcheck-$version.zip &> /dev/null
+        rm /tmp/shellcheck-$version.zip &>/dev/null
         echo "Unable to move shellcheck to /usr/bin"
         return 1
     fi
 
-    rm /tmp/shellcheck-$version.zip &> /dev/null
+    rm /tmp/shellcheck-$version.zip &>/dev/null
 
     return 0
 }
 
 if [ "${OS}" = "linux" ]; then
-    if ! which sudo > /dev/null; then
+    if ! which sudo >/dev/null; then
         "$SCRIPTPATH/install_linux_deps.sh"
     else
         sudo "$SCRIPTPATH/install_linux_deps.sh"
     fi
 elif [ "${OS}" = "darwin" ]; then
     if [ "${CIRCLECI}" != "true" ]; then
         brew update
-        brew tap homebrew/cask
+       if [[ $(brew --version | head -1 | cut -d' ' -f2) < "2.5.0" ]]; then
```

Comment:
also I don't think you can do semantic version checks with just bash lessthan.

Reply:
You are correct @bbroder-algo ! I gave it a thorough dive and made it a permanent useful small util (compare_semantic_versions) which considers a complete semantic versioning numeric and ordered comparison (MAJOR-->MINOR--->PATCH) and the test is included in the script and commented! Now it works like a charm and in the correct way (re-usable anywhere in the project)